### DISCUSSION TITLE
Google setting log no longer containing indicators of working app

### DISCUSSION THREAD

**Discussion by LeCanardQuoi** [cds-snc/covid-alert-documentation#57] [Q&A] [answered]
# Summary | Résumé

The Google setting log used to show on a regular basis the number of exposure checks from diagnosed users reporting back Canada wide. It was of limited interest but at least provided the app user with a sense that the app was working. There is no longer any "heartbeat" indication of a working app, at least for someone from a province that has not signed on yet (BC). Significant app events like activating/deactivating the app, other commands  e.g. setting/changing parameters: province/language, proximity timer/signal strength threshold, on and off quarantine cds-snc/covid-alert-app#1380, deleting random keys, chat/messaging alert cds-snc/covid-alert-documentation#50, and turning on and off the Bluetooth should all be logged, alongside proximity contacts cds-snc/covid-alert-app#1304 and QR code scan cds-snc/covid-alert-documentation#44.

# Steps to reproduce | Étapes à reproduire

Configure (BC or Alberta), access Google Setting log via Setting, Google, Covid Alert, inspect log and confirm absence of "heartbeat" app event

# Environment | Environnement logiciel

> Latest Android&device&app release.

# Others

Requirements for this Google log setting are unclear. Many proposed enhancements requests wish to add more useful info for the benefit of app core users, testers, troubleshooting and potentially contact tracing people via cds-snc/covid-alert-documentation#50.

**Comment by timarney** [comment, marked as answer]
Believe they moved it to the "more menu" i.e. "the three dot menu" top right

<img width="400" src="https://user-images.githubusercontent.com/62242/109003758-d44d0280-7675-11eb-9ea6-a948eab4a429.jpg" >

**Comment by obrien-j** [comment]
In keeping with our newly really release cds-snc/covid-alert-app#31 practices, I've moved this up into the covid-alert-documentation repo as our central point for taking input from folks outside the team, and am converting the issue into a Discussion.
